### ISSUE TITLE
Add ESLint And Config

### ISSUE THREAD

**Issue by alexfinnarn** [CUCentralAdvancement/essential-cu#21]
https://github.com/CUCentralAdvancement/giving-frontend/blob/master/.eslintrc.json

Should be able to add all that verbatim. Includes A11Y checkers.

**Comment by alexfinnarn** [comment]
duplicate of #41
